### PR TITLE
Added manifest.json

### LINKED ISSUE #1219: Progressive Web App: Create a web-app Manifest
The web app manifest provides information about an application (such as name, author, icon, and description) in a JSON text file. The purpose of the manifest is to install web applications to the home screen of a device, providing users with quicker access and a richer experience. Web app manifests provide the ability to save a site bookmark to a device's home screen. Though we can use a web app manifest on any site, they are required for progressive web apps.

### PR DESCRIPTION
**Pull request by jajodiaraghav**
Fixes #1219 

Changes: [Add here what changes were made in this issue and if possible provide links.]
Added manifest.json for webapp

@Princu7 Can you please guide me where should I place the manifest.json file so that it can get generated along with other files of the webapp?

### CODE REVIEW

**Comment by jajodiaraghav** [comment]
Pinging @Princu7 @geekyd

**Comment by jajodiaraghav** [comment]
Hi @geekyd I haven't made any changes to the app generator. I am not sure how generate a manifest.json file inside every app generated. A little hint might help :)

**Comment by geekyd** [comment]
@jajodiaraghav We copy the entire www folder while generating an application. So we just need to check if the generated application contains the manifest or not. Just generate an application and also share link of the generator here for test. :)

**Comment by jajodiaraghav** [comment]
Thanks for the info @geekyd 
Link: https://open-event-webapp.herokuapp.com/
I tested the app locally but there was no manifest.json file found in the generated application. Am I putting it in the wrong folder? Or do I have to do some modification in the backend folder?
I would be glad if you could help, I am new to this repo :)

File: src/www/manifest.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
+{
```
Comment (geekyd) [inline src/www/manifest.json]: @jajodiaraghav Add this to `src/backend/assets/`

**Comment by jajodiaraghav** [comment]
@geekyd PR updated. Please check the deployment again. It works perfectly fine for me. We also get the manifest file :)

**Comment by geekyd** [comment]
@jajodiaraghav Is the deployment link updated? For me it fails at 70% regardless of the sample being used.

**Comment by jajodiaraghav** [comment]
This is really weird. I tested in twice and it worked fine.

**Comment by geekyd** [comment]
@jajodiaraghav I'm still not able to generate web app. Can you please check. Also mention the sample you are using for generation.

**Comment by jajodiaraghav** [comment]
@geekyd I do not know why it is failing at your end. Here is a video showing that it works flawlessly. I have used FOSSASIA16.zip in the video, but also tested it with some other samples.

ezgif com-video-to-gif

**Comment by geekyd** [comment]
@jajodiaraghav Please check this
screen shot 2017-11-27 at 9 14 33 pm

Can you check the heroku logs to find out what is happening

**Comment by jajodiaraghav** [comment]
Link to video: https://vimeo.com/244667823

Also, The logs aren't helpful :(
capture

**Pull request closed without merging**